### PR TITLE
Update Update-M365TeamsApp.md

### PR DESCRIPTION
**Pull request by satish-timiri**

### CODE REVIEW

File: teams/teams-ps/teams/Update-M365TeamsApp.md [hunk 1]
```diff
@@ -33,11 +33,25 @@ This cmdlet allows administrators to modify app state and availability by adding
 ### Example 1
 
 ```powershell
-PS C:\> Update-M365TeamsApp -Id 3acac8f6-4e9e-4ee1-9c0e-0ea837d15ea3 -IsBlocked $true -AppAssignmentType UsersAndGroups -OperationType Add -Users eec823bd-0979-4cf8-9924-85bb6ffcb57d -Groups 37da2d58-fc14-453e-9a14-5065ebd63a1d
+PS C:\> Update-M365TeamsApp -Id 4c4ec2e8-4a2c-4bce-8d8f-00fc664a4e5b -AppAssignmentType Everyone
 ```
+Updates availablity assignment information of 4c4ec2e8-4a2c-4bce-8d8f-00fc664a4e5b app to Everyone.
```
Comment (marina-hayrapetyan) [inline teams/teams-ps/teams/Update-M365TeamsApp.md:38]: Updates the availability value for Bookings app (App ID 4c4ec2e8-4a2c-4bce-8d8f-00fc664a4e5b) to Everyone.

File: teams/teams-ps/teams/Update-M365TeamsApp.md [hunk 2]
```diff
@@ -33,11 +33,25 @@ This cmdlet allows administrators to modify app state and availability by adding
...
 
+### Example 2
+
+```powershell
+PS C:\> Update-M365TeamsApp -Id 4c4ec2e8-4a2c-4bce-8d8f-00fc664a4e5b -IsBlocked $true -AppAssignmentType UsersAndGroups -OperationType Add -Users eec823bd-0979-4cf8-9924-85bb6ffcb57d, eec823bd-0979-4cf8-9924-85bb6ffcb57e -Groups 37da2d58-fc14-453e-9a14-5065ebd63a1d, 37da2d58-fc14-453e-9a14-5065ebd63a1e, 37da2d58-fc14-453e-9a14-5065ebd63a1b, 37da2d58-fc14-453e-9a14-5065ebd63a1f, 37da2d58-fc14-453e-9a14-5065ebd63a1a
+```
 Updates state and availablity assignment information of 3acac8f6-4e9e-4ee1-9c0e-0ea837d15ea3 app.
```
Comment (marina-hayrapetyan) [inline teams/teams-ps/teams/Update-M365TeamsApp.md:45]: Unblocks CSP Customer App (App ID 4c4ec2e8-4a2c-4bce-8d8f-00fc664a4e5b) and updates availability setting for the app to include 2 users and 5 groups.

File: teams/teams-ps/teams/Update-M365TeamsApp.md [hunk 3]
```diff
@@ -33,11 +33,25 @@ This cmdlet allows administrators to modify app state and availability by adding
...
 
+### Example 3
+
+```powershell
+PS C:\> Update-M365TeamsApp -Id 4c4ec2e8-4a2c-4bce-8d8f-00fc664a4e5b -IsBlocked $true
+```
+Unblocks 4c4ec2e8-4a2c-4bce-8d8f-00fc664a4e5b app.
+
```
Comment (marina-hayrapetyan) [inline teams/teams-ps/teams/Update-M365TeamsApp.md:53]: Unblocks Bookings app (App ID 4c4ec2e8-4a2c-4bce-8d8f-00fc664a4e5b).

File: teams/teams-ps/teams/Update-M365TeamsApp.md (hunk 1 above)
Comment (satish-timiri) [reply teams/teams-ps/teams/Update-M365TeamsApp.md]: Updated as requested

File: teams/teams-ps/teams/Update-M365TeamsApp.md (hunk 3 above)
Comment (satish-timiri) [reply teams/teams-ps/teams/Update-M365TeamsApp.md:53]: Updated as requested

File: teams/teams-ps/teams/Update-M365TeamsApp.md (hunk 2 above)
Comment (satish-timiri) [reply teams/teams-ps/teams/Update-M365TeamsApp.md]: Updated as requested

**Comment by michbrown-png** [comment]
@satish-timiri - Approved. I will provide sign-off after @marina-hayrapetyan agrees to the changes

**Review by marina-hayrapetyan — APPROVED** [review]

**Pull request merged**